app/main.py

```diff
@@ -1,6 +1,25 @@
+import requests
+import os
+
+
+weather_site = "https://api.weatherapi.com/v1/current.json?key="
```

Comment:
Read the checklist:

![image](https://user-images.githubusercontent.com/104218550/213658035-9fa83b2f-0817-4c23-92ba-f6a2a2bf51dc.png)